department-of-veterans-affairs/va.gov-cms
# VACMS-11688 Add exemption for Lovell both listing page paths.

## Description

Closes #11688 

## Testing done


## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As an admin
1. go here https://pr11702-jcznngjlnhe7bu3oeenvtmqm3iwjqa8b.ci.cms.va.gov/admin/content/bulk?title=&type=All&moderation_state=All&taxonomy_entity_index_tid_depth=347
2. Select all nodes, and choose the action to publish latest revision.
3.  Assert that we do not have duplication of listing pages
   -  Go here https://pr11702-jcznngjlnhe7bu3oeenvtmqm3iwjqa8b.ci.cms.va.gov/admin/content/bulk?title=&type=event_listing&moderation_state=All&taxonomy_entity_index_tid_depth=347 
   - [ ] Validate that all three items end in /events/ with no -0
   -  go here https://pr11702-jcznngjlnhe7bu3oeenvtmqm3iwjqa8b.ci.cms.va.gov/admin/content/bulk?title=&type=press_releases_listing&moderation_state=All&taxonomy_entity_index_tid_depth=347
   - [ ] validate that all three items end in 'news-releases' with no -0
   -  go here https://pr11702-jcznngjlnhe7bu3oeenvtmqm3iwjqa8b.ci.cms.va.gov/admin/content/bulk?title=&type=story_listing&moderation_state=All&taxonomy_entity_index_tid_depth=347
   - [ ] validate that all three items end in 'stories' with no -0
4. Assert that non-listing pages function as they should
   - Go to https://pr11702-jcznngjlnhe7bu3oeenvtmqm3iwjqa8b.ci.cms.va.gov/lovell-federal-va-health-care/about-us  
   - [ ] validate that page exists
   - Go to https://pr11702-jcznngjlnhe7bu3oeenvtmqm3iwjqa8b.ci.cms.va.gov/lovell-federal-tricare-health-care/about-us  
   - [ ] validate that it redirects to 'ovell-federal-va-health-care/about-us  ' meaning two aliases exist.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
